collector/ilm_status_response.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+// IlmStatusResponse is a representation of elasticsearch ilm status
+type IlmStatusResponse struct {
```

Comment:
This is such a small amount of data that it should be in the `ilm_status.go` file. I know that other collectors are broken out like this, but I'm trying to get away from that.